Changed the term whitelist to allowlist in Documentation (#200)
expressjs/cors: README.md

}

app.get('/products/:id', cors(corsOptions), function (req, res, next) {

>   res.json({msg: 'This is CORS-enabled for a allowlisted domain.'})

Reviewer: ```suggestion
  res.json({msg: 'This is CORS-enabled for an allowed domain.'})
```

Would this wording for for you? It would help move it that much further away from whitelist, I think.

Reviewer 2: This sounds perfect :100:

WDYT @jkasun?

}

app.get('/products/:id', cors(corsOptionsDelegate), function (req, res, next) {

>   res.json({msg: 'This is CORS-enabled for a allowlisted domain.'})

Reviewer: ```suggestion
  res.json({msg: 'This is CORS-enabled for an allowed domain.'})
```

Reviewer 2: /cc @jkasun
